packages/local-environment/hardhat/hardhat.config.js

```diff
@@ -1,4 +1,5 @@
 require("@nomiclabs/hardhat-waffle");
+require("@nomiclabs/hardhat-etherscan");
```

Comment:
Why is this needed again?

Reply:
Hmm I don't actually think that's necessary -  was just there. Removing.